src/report.js

```diff
@@ -12,7 +12,7 @@ export const generateReport = async (results) => {
     lines.push(`| File | Generated description |`);
     lines.push(`| :--- | :--- |`);
     results.forEach(r => {
-        lines.push(`| \`${r.page}\` | ${r.description} |`);
+        lines.push(`| \`${r.page}\` | ${r.description} | ${r.status} |`);
```

Comment:
Include the error message in this table if there is any. Make sure that the table stays consistent in format.